app/models/user.rb

```diff
@@ -3,4 +3,14 @@ class User < ApplicationRecord
   has_many :user_bets
   has_many :debts,   class_name: 'Debt', foreign_key: :debtor_id
   has_many :credits, class_name: 'Debt', foreign_key: :creditor_id
+
+  scope :winners, lambda {
```

Comment:
Curly braces are for single line blocks.  Multiline block require `do...end` 
also note that you will have to get fancy with parenthesis by moving to the do/end syntax.

for lambdas preferred syntax is: 
oneline:
`->{...}`

multiline:
```ruby
lambda do
...
end
```